extensions/wikia/TemplateDraft/TemplateConverter.class.php

```diff
@@ -102,4 +110,30 @@ public function createDataTag( $source, $label = '' ) {
 		}
 		return "\t<data source=\"{$source}\"><label>{$label}</label></data>\n";
 	}
+
+	public function generatePreviewSection( $variables ) {
+		$return = "== Usage & preview ==\n";
+		$return .= "<noinclude>\n";
```

Comment:
The `<noinclude>` go before the section heading so the section heading is also removed from transclusions.